pkg/node-servant/static-pod-upgrade/upgrade.go

```diff
@@ -0,0 +1,172 @@
+/*
+Copyright 2023 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"k8s.io/klog/v2"
+
+	"github.com/openyurtio/openyurt/pkg/node-servant/static-pod-upgrade/util"
+)
+
+const (
+	DefaultUpgradeDir                   = "openyurtio-upgrade"
+	DefaultStaticPodRunningCheckTimeout = 2 * time.Minute
```

Comment:
how about add a new parameter for upgrade command to specify static pod check timeout? because sometimes two minutes maybe not enough.

Reply:
done